taxi/tests/test_forms.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTestCase(TestCase):
+    def test_driver_creation_form_with_license_number_first_last_name(self):
+        test_data = {
+            "username": "test_user",
+            "license_number": "NNN12345",
+            "first_name": "John",
+            "last_name": "Doe",
+            "password1": "<PASSWORD>",
+        }
+
+        form = DriverCreationForm(data=test_data)
+        self.assertFalse(form.is_valid())
+        self.assertEqual(form.cleaned_data, test_data)
```

Comment:
The assertion `self.assertEqual(form.cleaned_data, test_data)` will fail if the form is not valid, as `cleaned_data` is only available when the form is valid. Consider checking `form.errors` if you expect the form to be invalid.